commands/transfers/transferList.js

```diff
@@ -214,38 +265,96 @@ export const unlistCmd = {
+                },
+                {
+                    type: 3,
+                    name: 'position1',
+                    description: 'Primary position you can play',
+                    required: true,
+                    choices: [
+                        { name: 'GK', value: 'GK' },
+                        { name: 'LB', value: 'LB' },
+                        { name: 'CB', value: 'CB' },
+                        { name: 'RB', value: 'RB' },
+                        { name: 'CM', value: 'CM' },
+                        { name: 'LW', value: 'LW' },
+                        { name: 'RW', value: 'RW' },
+                        { name: 'ST', value: 'ST' }
+                    ]
+                },
+                {
+                    type: 3,
+                    name: 'position2',
+                    description: 'Secondary position you can play',
+                    required: false,
+                    choices: [
+                        { name: 'GK', value: 'GK' },
+                        { name: 'LB', value: 'LB' },
+                        { name: 'CB', value: 'CB' },
+                        { name: 'RB', value: 'RB' },
+                        { name: 'CM', value: 'CM' },
+                        { name: 'LW', value: 'LW' },
+                        { name: 'RW', value: 'RW' },
+                        { name: 'ST', value: 'ST' }
+                    ]
+                },
+                {
+                    type: 3,
+                    name: 'position3',
+                    description: 'Tertiary position you can play',
+                    required: false,
+                    choices: [
+                        { name: 'GK', value: 'GK' },
+                        { name: 'LB', value: 'LB' },
+                        { name: 'CB', value: 'CB' },
+                        { name: 'RB', value: 'RB' },
+                        { name: 'CM', value: 'CM' },
+                        { name: 'LW', value: 'LW' },
+                        { name: 'RW', value: 'RW' },
+                        { name: 'ST', value: 'ST' }
+                    ]
+                },
+                {
+                    type: 3,
+                    name: 'extra_info',
+                    description: 'Additional information (e.g., "only looking for Div 1 teams")',
+                    required: false
+                }
+            ]
+        },
+        {
+            name: 'remove',
+            type: 1,
+            description: 'Remove yourself from the LFT list'
+        }
+    ]
+};
+
+const lftHandler = async (params) => {
+    const { options } = params;
+    const subcommand = options[0].name;
+
+    if (subcommand === 'add') {
+        return lftAdd({ ...params, options: options[0].options });
+    } else if (subcommand === 'remove') {
+        return lftRemove(params);
+    }
 };
 
-export default [transferListCmd, unlistCmd, lftCmd];
+export default [transferListCmd, unlistCmd, { ...lftCmd, func: lftHandler }];
```

Comment:
I would recommend you to check the starting6 starting8 in lineup.js to see how to handle multi commands :)